File: backend/components/metiers/administration_publique/guides/fallback/services/fallback_services.js
// guides/fallback/fallback_services.js – Fallbacks et tests avancés pour guides services

function fallbackServiceConfig() {
  // Fallback de configuration service pour tests d’intégration
  return {
    name: 'service-fallback',
    endpoint: '/api/fallback',
    description: 'Fallback de test pour service',
  };
}

module.exports = { fallbackServiceConfig };

File: backend/components/metiers/administration_publique/guides/fallback/services/fallback_services.py
# guides/fallback/fallback_services.py – Fallbacks et tests avancés pour guides services

def fallback_service_config():
    # Fallback de configuration service pour tests d’intégration
    return {
        'name': 'service-fallback',
        'endpoint': '/api/fallback',
        'description': 'Fallback de test pour service',
    }
